org/general/check-assignments-of-devices.ps1

```diff
@@ -0,0 +1,286 @@
+<#
+  .SYNOPSIS
+  Check Intune assignments for a given (or multiple) Device Names.
+

```

Comment:
Warum über den Device name und nicht im Device-Kontext in RJ?

Reply:
Weil viele?

Reply:
Erstes Ziel war ein Allgemeines Runbook für mehrere Gruppen. Vielleicht erstelle ich noch eins im jeweiligen Kontext.